src/modules/Dashboard/components/TakeNowModal/TakeNowModal.tsx

```diff
@@ -33,6 +45,18 @@ const ALLOWED_TEAM_MEMBER_ROLES: readonly Roles[] = [
 type AnyTeamSearchType = 'team' | 'any-participant';
 type FullTeamSearchType = 'team' | 'full-participant';
 
+const getAccountType = (subject: ParticipantDropdownOption | null) => {
+  if (!subject) return null;
+
+  if (subject.userId) {
```

Comment:
I agree it would be nice to distinguish, but there would be some extra lift to attach pending status to all places where Take Now is initiated. I'll make a note of it in the ticket, which will be reviewed by the data team and they can decide whether the distinction is important enough to create a follow-up task.